typescript/infra/scripts/warp-routes/token_config.ts

```diff
@@ -0,0 +1,66 @@
+import { ChainMap } from '@hyperlane-xyz/sdk';
+
+interface NativeTokenConfig {
+  chainId: number;
+  symbol: string;
+  name: string;
+  type: 'native';
+  decimals: number;
+  hypNativeAddress: string;
+  protocolType?: 'ethereum' | 'solana';
+}
+
+interface CollateralTokenConfig {
+  // Typescript does not allow literal value value 'collateral' even if it matches the enum's value
+  type: 'collateral';
+  address: string;
+  chainId: number;
+  decimals: number;
+  symbol: string;
+  name: string;
+  hypCollateralAddress: string;
+  isSpl2022?: boolean;
+  protocolType?: 'ethereum' | 'solana';
+}
+
+export type WarpTokenConfig = ChainMap<
+  CollateralTokenConfig | NativeTokenConfig
+>;
+
+export const tokenList: WarpTokenConfig = {
+  // bsc
+  bsc: {
+    type: 'collateral',
```

Comment:
The `scripts` folder probably isn't the right place for hard-coded configs/consts like these :)
Do these belong in `src/config` somewhere?

Reply:
I was the one who said hard coding to start is fine, but might be nice to still put it in src/config and just import from there